src/server/pfs/server/url.go

```diff
@@ -26,6 +24,10 @@ const (
 	defaultURLTaskSize = 1000
 )
 
+var (
+	threshold = int64(1000000) // threshold is overridden when testing. Default is 1 Mb
```

Comment:
1MB is too low for a task, the overhead of managing the tasks will outweigh the work being done. Also, a file set for every MB would be excessive in terms of compaction. I think between 100MB - 1GB would be the right range for this, so that the task is on the order of seconds. I would lean towards 1GB to give it parity with the sharding we do in compaction.